### ISSUE TITLE
The description is left Justified even with `<center>`

### ISSUE THREAD

**Issue by kingabzpro** [gradio-app/gradio#840]
### Describe the bug

I think the new update of Gradio broke description on HF. You can check all the apps. I am using center html. Other apps are also effected.

### Reproduction

Here is the app link: https://huggingface.co/spaces/kingabzpro/Urdu-ASR-SOTA

### Screenshot

![image](https://user-images.githubusercontent.com/36753484/159069496-252da284-c5e2-4e7d-a189-82f86722f473.png)

### System Info

```shell
I am using huggingface spaces environment and Edge as browser.
```

### Severity

annoyance

**Comment by abidlabs** [comment]
Hi @kingabzpro, thanks for reporting this issue. This has been fixed in the latest version of gradio, `2.8.12`. 

I think if you update the `sdk_version` in your README file: https://huggingface.co/spaces/kingabzpro/Urdu-ASR-SOTA/blob/main/README.md, it should be fixed!

**Comment by kingabzpro** [comment]
It worked.😎
